Services/Mail/classes/class.ilFormatMail.php

```diff
@@ -102,7 +102,7 @@ public function formatLinebreakMessage(string $message): string
     {
         $formatted = [];
 
-        $linebreak = $this->mail_options->getLinebreak();
+        $linebreak = ilMailOptions::DEFAULT_LINE_BREAK;
```

Comment:
@fhelfer Maybe this is causing the errors Oliver reported You have to remove **all** linebreak related code.